### PR TITLE
Update remove-licenses-from-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the section for searching disconnected or de-licensed mailboxes.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23798

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/remove-licenses-from-users.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ When you use the **Active users** page to unassign licenses, you unassign produc
...
 - Files saved in OneDrive for Business aren't deleted unless the user is deleted from the Microsoft 365 admin center or is removed through Active Directory synchronization. For more information, see [OneDrive retention and deletion](/onedrive/retention-and-deletion).
-- When the license is removed, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or eDiscovery (Premium). For more information, see "Searching disconnected or de-licensed mailboxes" in [Content Search in Microsoft 365](../../compliance/ediscovery-content-search.md).
+- When the license is removed, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or eDiscovery (Premium). For more information, see "Searching disconnected or de-licensed mailboxes" in [Content Search in Microsoft 365](/compliance/ediscovery-content-search-reference?view=o365-worldwide#searching-disconnected-or-de-licensed-mailboxes).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/remove-licenses-from-users.md]:
```suggestion
- When the license is removed, the user's mailbox is no longer searchable by using an eDiscovery tool such as Content Search or eDiscovery (Premium). For more information, see "Searching disconnected or de-licensed mailboxes" in [Content Search in Microsoft 365](/microsoft-365/compliance/ediscovery-content-search-reference#searching-disconnected-or-de-licensed-mailboxes).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Comment by cmcatee-MSFT** [comment]
"For more information" sentence was removed from the article.

**Pull request closed without merging**